tailwind.config.js

```diff
@@ -46,6 +46,14 @@ module.exports = {
       lg: '1024px',
       xl: '1280px',
       '2xl': '1536px',
+    },
+    extend: {
+      colors: {
+        raTheme: {
+          scrollthumb: 'var(--link-color)',
+          scrolltrack: 'var(--text-color-muted)'
```

Comment:
Scrolltrack seems to be overly exposed on light scheme

Reply:
Agreed, after changing `scrollthumb` to use `--text-color`, this seems a bit better, however I am very finicky about my monitor's color calibration and it is entirely possible we are not seeing the same thing. Attached is a screenshot. Do you still find this difficult to see?

![Screenshot 2023-03-02 at 6 56 32 PM](https://user-images.githubusercontent.com/3984985/222607066-3b6d242c-08d8-4a2a-b8eb-2ec007676a98.png)

Generally, _everything_ using this combination of scheme and color settings appears a bit washed to my eyes.